cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -1103,6 +1103,11 @@
     /** Key to specify the kerberos principal of the entity owner **/
     public static final String PRINCIPAL = "principal";
 
+    public static final class Internal {
```

Comment:
I am not sure if we need to introduce this `Internal` section with just a single config

Reply:
Fixed